test/splitting/strings.stmt

```diff
@@ -53,4 +53,24 @@ someMethod("""first line does not fits here
 <<<
 someMethod(
     """first line does not fits here
-""");
\ No newline at end of file
+""");
+>>> format multiline as a block with a leading argument
+someMethod("foo", """
+  some
+  text
+""");
+<<<
+someMethod("foo", """
+  some
+  text
+""");
+>>> format multiline a block with a trailing argument
+someMethod("""
+  some
+  text
+""", "foo");
+<<<
+someMethod("""
+  some
+  text
+""", "foo");
```

Comment:
Can you test:

* Some negative cases, like a multiline string with non-block arguments before and after.
* A sequence of multiline strings in an argument list.
* A mixture of multiline strings and other block arguments like collection literals.